backend/vercel/vercel.go

```diff
@@ -285,7 +285,9 @@ func HandleLog(w http.ResponseWriter, r *http.Request) {
 			http.Error(w, err.Error(), http.StatusBadRequest)
 			return
 		}
-		logs = append(logs, l)
+		if l.Message != "" {
```

Comment:
Do we know why this was actually happening?

What I'm trying to figure out is whether a log without a message is actually valid (e.g. one only wanted to use fields).

I did confirm that logrus won't output anything without a message, e.g. this does nothing:

```go
import (
	log "github.com/sirupsen/logrus"
)
log.WithContext(ctx).WithField("foo", "bar")
```

Mostly wondering if you were able to replicate this and why vercel was sending us logs without a message in the first place.

Reply:
Oh wait...I can do this:

```go
log.WithContext(ctx).WithField("foo", "bar").Info()
```

![Screenshot 2023-04-14 at 9 24 40 AM](https://user-images.githubusercontent.com/58678/232086873-c1288fb3-df1c-4b08-b82f-d3ba6a5cea7b.png)

Reply:
@Vadman97 - do you think we could bring this up with a team on whether or not we consider these types of logs to be valid?

Reply:
Hmm, there might be some log drain data that has no message body. maybe other events like a deployment build workflow sends empty log messages populating other metadata fields? https://vercel.com/docs/concepts/observability/log-drains-overview/log-drains-reference#format-and-transport

I'll shoot them an email about this but in the meantime, I suggest that we merge this with a specific vercel warning about the empty messages we'd be dropping.